nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/pom.xml

```diff
@@ -26,11 +26,19 @@
         performs any decryption/retrieval of sensitive configuration properties.
     </description>
     <packaging>jar</packaging>
+    <properties>
+        <aws-java-sdk-version>1.11.461</aws-java-sdk-version>
+    </properties>
     <dependencies>
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-properties</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-kms</artifactId>
+            <version>${aws-java-sdk-version}</version>
+        </dependency>
```

Comment:
Just a minor note that we'll need to consider if anything needs to be added to the LICENSE/NOTICE in nifi-assembly and in nifi-framework-nar, for this new dependency and any new transitive dependencies it brings in.

Reply:
I don't know if anything needs to be added or not, but I do know this dependency already exists in `nifi-nar-bundles/nifi-aws-bundle`.  Is that LICENSE/NOTICE sufficient, or do we need something else?

Reply:
In that case, it likely means the assembly's LICENSE/NOTICE is already correct (assuming whoever setup the AWS NAR did the right thing and bubbled up anything that is necessary). So if there is even anything to do, it would likely only be in the framework NAR's LICENSE/NOTICE.